photography-site-demo.js/.config/index.js

```diff
@@ -1,10 +1,12 @@
 'use strict';
 
 require('dotenv').config();
-if (process.env.NODE_ENV === 'development') {
-    console.log('using development');
-    module.exports = require('./development');
-} else {
-    console.log('using production');
-    module.exports = require('./production');
+if (process.env.NODE_ENV) {
```

Comment:
Same suggestions as in #23: if we're going to use dotenv, please use dotenv, no need to use the `.config` pattern. The .config pattern is a way to use config files rather than environment variables, and I'd rather not mix the two.

Reply:
@vkarpov15 I'll have this app and other two sample apps fixed similar to the Netlify ecommerce and ll submit a PR

Reply:
Thanks! @vkarpov15 
@kathirsvn  I have already made corresponding changes for photography demo app. Thanks!